### DISCUSSION TITLE
I played a little bit with the js codes of the application :D

### DISCUSSION THREAD

**Discussion by safak45x** [pear-devs/pear-desktop#1261] [Ideas]
![Screenshot_20210915_054934](https://user-images.githubusercontent.com/59151376/133362794-0cdacf3b-0398-4d7c-9e80-d1167d63e397.png)
![Screenshot_20210915_055404](https://user-images.githubusercontent.com/59151376/133362800-0bf73229-87f4-4520-975b-4a4058e02b56.png)

plugin etc all works very well

**Comment by lg-nineoneninethreenine** [comment]
Hey... @th-ch can't we create a YouTube client based on this? I mean, the PWA needs a browser to be open, but if this is made, you don't need a browser open at all! Maybe someone can help with the icon, a high-quality icon can do, right?
